edx/edx-ora2
# fix(rubric reuseplaceholder message)

**TL;DR -** I changed a variable name in one place and forgot to change it in another

JIRA: [AU-22](https://openedx.atlassian.net/browse/AU-22)

**What changed?**
-  whoops

**Testing Instructions**

- Install ORA, enable flag (openresponseassessment.rubric_reuse) go to a course with other ORAs and add another ORA.
- Before the change, the rubric reuse will work correctly, but the placeholder on the input says that there aren't any other ORAs in the course. You also won't be able to click on the input (but the button will still show the dropdown)
- After the change, it shouldn't show that message, and you should be able to type
- But if you go to a blank course and create an ORA, it should say that there are no ORAs in the course
**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
